### ISSUE TITLE
Downlink Data not received by node 

### ISSUE THREAD

**Issue by omaralaoui1999** [CongducPham/LowCostLoRaGw#308]
![capture](https://user-images.githubusercontent.com/76405361/117546682-223d9980-b01b-11eb-8f22-4955fc4efcdd.PNG)
![downlink](https://user-images.githubusercontent.com/76405361/117546683-236ec680-b01b-11eb-81d6-03d987a418d6.PNG)

**Comment by zaiplanet** [comment]
Can we send downlink reset node message from gateway to node and when node revieced message will reset node?

**Comment by CongducPham** [comment]
Yes, it is up to the device to interpret the command and to initiate the reset procedure.
